README.md

```diff
@@ -99,6 +99,123 @@ Run the following command:
 docker-compose up -d
 ```
 
+#### Docker Swarm w/ Docker Secrets:
+
+This section discusses running an instance of ibeam inside a locked docker swarm, and using the docker swarm facilities for managing secrets.
+Please refer to the following articles for in-depth details on docker swarm, locking the swarm, and using docker secrets.
+
+- [Swarm mode overview](https://docs.docker.com/engine/swarm/)
+- [Lock your swarm to protect its encryption key](https://docs.docker.com/engine/swarm/swarm_manager_locking/)
+- [Manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/)
+
+Once you have a locked docker swarm instance initialized, you can create the secrets.
+On your host system create two secure (meaning not world-readable) files containing your Interactive Brokers account name and password:
+
+1. ib.account.txt
+2. ib.password.txt
+
+Next, inject these secrets into the docker swarm by using `docker secret create`:
+
+```posh
+docker secret create IBEAM_ACCOUNT_v1 ib.account.txt
+docker secret create IBEAM_PASSWORD_v1 ib.password.txt
+```
+
+Once you've initialized the secrets you should remove the original files from your host system.
+
+Create a `docker-compose.yml` file that references the new secrets, and that configures its environment to pull the secrets from the container's filesystem:
+
+```yaml
+version: "3.7"
+
+secrets:
+  IBEAM_ACCOUNT_v1:
+    external: true
+  IBEAM_PASSWORD_v1:
+    external: true
+
+services:
+  ibeam:
+    image: "voyz/ibeam"
+    environment:
+      IBEAM_SECRETS_SOURCE: "fs"
+      IBEAM_ACCOUNT: "/run/secrets/IBEAM_ACCOUNT"
+      IBEAM_PASSWORD: "/run/secrets/IBEAM_PASSWORD"
+    secrets:
+      - source: "IBEAM_ACCOUNT_v1"
+        target: "/run/secrets/IBEAM_ACCOUNT"
+        uid: "1000"
+        gid: "1000"
+        mode: 0400
+      - source: "IBEAM_PASSWORD_v1"
+        target: "/run/secrets/IBEAM_PASSWORD"
+        uid: "1000"
+        gid: "1000"
+        mode: 0400
+    volumes:
+      - type: "bind"
+        source: "conf.yaml"
+        target: "/srv/clientportal.gw/root/conf.yaml"
+    ports:
+      - target: 5000
+        host_ip: "127.0.0.1"
+        published: 5000
+        protocol: "tcp"
+        mode: "host"
+```
+
+The `conf.yaml` file that is referenced in the volumes block is a copy of the `root/conf.yaml` file found in the the
+[clientportal.gw.zip](http://download2.interactivebrokers.com/portal/clientportal.gw.zip)
+downloaded from
+[Interactive Brokers Client Portal Web API Gateway](https://interactivebrokers.github.io/cpwebapi/)
+(or see the copy from
+[copy_cache/clientportal.gw/root](https://github.com/Voyz/ibeam/tree/master/copy_cache/clientportal.gw/root),
+which is embedded in the voyz/ibeam docker image.
+
+Toward the end of the `conf.yaml` there is a block to define IPs to trust and reject, e.g.,
```

Comment:
This information on IPs is actually helpful, however isn't it agnostic to which secrets method is used? I think it could be added here instead: https://github.com/Voyz/ibeam/wiki/Gateway-Configuration

Reply:
There's specifically a discussion I wanted to highlight about needing to add the docker_gwbridge IP address.  Would you review my changes and see if you still think the discussion should be moved?

Reply:
I think it is justified in its current format 👍